From: Juergen Beisert <jbe@pengutrinix.de>
Date: Fri, 14 Mar 2014 13:00:15 +0100
Subject: [PATCH] fix buildsystem

Use the correct built-in variables 'make' uses to compile C code and also
make it DESTDIR aware.

Signed-off-by: Juergen Beisert <jbe@pengutrinix.de>
---
 Makefile |   18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/Makefile b/Makefile
index a522607..369b149 100644
--- a/Makefile
+++ b/Makefile
@@ -1,15 +1,15 @@
-CFLAGS		+=	-O3 -lbz2
-
-PREFIX		?=	/usr/local
-INSTALL_PROGRAM	?=	${INSTALL} -c -s -m 555
-INSTALL_MAN	?=	${INSTALL} -c -m 444
+CFLAGS		+= -O3
+LDLIBS		+= -lbz2
+PREFIX		?= /usr/local
+INSTALL_PROGRAM	?= $(INSTALL) -c -s -m 555
+INSTALL_MAN	?= $(INSTALL) -c -m 444
 
 all:		bsdiff bspatch
 bsdiff:		bsdiff.c
 bspatch:	bspatch.c
 
 install:
-	${INSTALL_PROGRAM} bsdiff bspatch ${PREFIX}/bin
-.ifndef WITHOUT_MAN
-	${INSTALL_MAN} bsdiff.1 bspatch.1 ${PREFIX}/man/man1
-.endif
+	${INSTALL_PROGRAM} bsdiff bspatch $(DESTDIR)${PREFIX}/bin
+ifndef WITHOUT_MAN
+	${INSTALL_MAN} bsdiff.1 bspatch.1 $(DESTDIR)${PREFIX}/man/man1
+endif
